packages/build-tools/src/buildErrors/userErrorHandlers.ts

```diff
@@ -179,6 +179,38 @@ You are seeing this error because either:
+    // warning: Run script build phase '[CP-User] Generate app.manifest for expo-updates' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXUpdates' from project 'Pods')
+    // warning: Run script build phase 'Start Packager' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'testapp' from project 'testapp')
+    // warning: Run script build phase 'Bundle React Native code and images' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'testapp' from project 'testapp')
+    // warning: Run script build phase '[CP-User] Generate app.config for prebuilt Constants.manifest' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXConstants' from project 'Pods')
+    // /Users/expo/workingdir/build/managed/ios/Pods/Pods.xcodeproj: error: Signing for "EXUpdates-EXUpdates" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'EXUpdates-EXUpdates' from project 'Pods')
+    regexp: /error: Signing for "[a-zA-Z-0-9_]+" requires a development team/,
+    createError: (_, { job }) =>
+      job.type === Workflow.MANAGED
+        ? new UserFacingError(
+            'XCODE_CODE_SIGNING_ERROR',
+            `Starting from Xcode 14, resource bundles are signed by default, which requires setting the development team for each resource bundle target.
+To resolve this issue downgrade to the older Xcode version using an "image" field in eas.json, or upgrade to SDK 46 or higher.`,
+            'https://docs.expo.dev/build-reference/infrastructure/#ios-build-server-configurations'
+          )
+        : new UserFacingError(
+            'XCODE_CODE_SIGNING_ERROR',
+            `Starting from Xcode 14, resource bundles are signed by default, which requires setting the development team for each resource bundle target.
+To resolve this issue downgrade to the older Xcode version using an "image" field in eas.json, or turn off signing resource bundles in your Podfile, as in this example https://github.com/expo/expo/pull/19095.`,
```

Comment:
```suggestion
To resolve this issue, downgrade to an older Xcode version using the "image" field in eas.json, or turn off signing resource bundles in your Podfile: https://expo.fyi/r/disable-bundle-resource-signing`,
```

i added support for redirects to fyi to make this msg a bit nicer. i think the code we link to here is more useful than the pr. to see how to do this in the future: https://github.com/expo/fyi/blob/main/redirects/disable-bundle-resource-signing

Reply:
That sounds like a good idea, thank you!